### ISSUE TITLE
Joining relationships intermediate node causes SQL Exception

### ISSUE THREAD

**Issue by kul2002il** [kirschbaum-development/eloquent-power-joins#90]
Code:
```PHP
$query = WeeklyBlog::query()->joinRelationship('worker');
if($conditions)
{
    $query->joinRelationship('worker.department');
}
return $query->toSql();
```

Output if $conditions = true:
```SQL
select `weekly_blog`.* from `weekly_blog`
inner join `worker` on `weekly_blog`.`worker_nick` = `worker`.`nick`
inner join `worker` on `weekly_blog`.`worker_nick` = `worker`.`nick`
inner join `department` on `worker`.`department_id` = `department`.`id`
```

I want get
```SQL
select `weekly_blog`.* from `weekly_blog`
inner join `worker` on `weekly_blog`.`worker_nick` = `worker`.`nick`
/* is no duplication */
inner join `department` on `worker`.`department_id` = `department`.`id`
```
How to get it?
Maybe there's a way to solve this (some kind `addJoinRelationship()`),
but it's not write in README.md.

Seems to be related to #37 and #62.

Sorry, if it's duplicate.

**Comment by luisdalmolin** [comment]
I'm going to look into it.

**Comment by luisdalmolin** [comment]
@kul2002il this should be fixed in version 2.6.2: https://github.com/kirschbaum-development/eloquent-power-joins/releases/tag/2.6.2

**Comment by kul2002il** [comment]
Yes, it make application work.
Thanks.
